modules/pipeline/providers/dbgc/dbgc.service.go

```diff
@@ -38,10 +41,17 @@ const (
 	etcdDBGCDLockKeyPrefix = "/devops/pipeline/dbgc/dlock/"
 )
 
-// remove ListenDatabaseGC and EnsureDatabaseGC these two methods，
+type dbgcService struct {
+	js   jsonstore.JsonStore
+	etcd *etcd.Store
+
+	dbClient *db.Client
+}
+
+// PipelineDatabaseGC remove ListenDatabaseGC and EnsureDatabaseGC these two methods，
 // these two methods will create a lot of etcd ttl, will cause high load on etcd
 // use fixed gc time, traverse the data in the database every day
-func (r *Reconciler) PipelineDatabaseGC(ctx context.Context) {
+func (r *dbgcService) PipelineDatabaseGC(ctx context.Context) {
```

Comment:
`r` is strange. change all receiver to `s`.

Reply:
done